libs/content/workspace/src/tab/svg_editor/mod.rs

```diff
@@ -95,6 +96,9 @@ impl SVGEditor {
 
         self.inner_rect = ui.available_rect_before_wrap();
 
+        ui.painter()
+            .rect(self.inner_rect, 0., ui.style().visuals.extreme_bg_color, Stroke::NONE);
```

Comment:
you can use `rect_filled()` instead of `rect()` to avoid specifying the default stroke.